ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -1494,8 +1513,40 @@ class PartCentredForceHolder {
                FlightGlobals.Vessels.Where(v => is_manageable(v) &&
                                                 !v.packed)) {
         foreach (Part part in vessel.parts.Where(PartIsFaithful)) {
-          if (part.torque != Vector3d.zero) {
-            part_id_to_intrinsic_torque_.Add(part.flightID, part.torque);
+          Vector3d parachute_torque = Vector3d.zero;
+          if (part == vessel.rootPart &&
+              is_parachuting_kerbal(vessel) &&
+              parachuting_kerbal_angular_velocities_.TryGetValue(
+                  vessel, out Vector3d old_angular_velocity)) {
+            // Instead of applying forces some distance from the centre of mass,
+            // or applying torques, EVA parachutes directly set the angular
+            // velocity of the Kerbal.  We register a torque equivalent to the
+            // change in angular momentum; failing to do so would cause the
+            // parachute to act on the linear motion but not the attitude,
+            // so that Kerbals would keep spinning under their parachute in
+            // stock, and would orient themselves head-down (and parachute-down)
+            // with FAR.  See #2607.
+            // To quote ferram4, “consider it all cursed”.
+            Vector3d Δω_world = part.rb.angularVelocity - old_angular_velocity;
```

Comment:
I think I would really prefer if this code was in C++.  It's a bit annoying to pass things through the interface and retype all the data, but we would have a better confidence that everything is correct, including when we maintain it.  It could live directly in `interface.cpp`, no point in pushing it to some class.

Reply:
Done. Considering how sloppy our handling of the MTS system was, I am not sure this change is less error-prone. I tried cleaning up the existing conversion constant.